packages/expo-camera/src/Camera.types.ts

```diff
@@ -250,8 +250,34 @@ export type Point = {
   y: number;
 };
 
+export type BarCodeSize = {
+  /**
+   * The height value.
+   */
+  height: number;
+  /**
+   * The width value.
+   */
+  width: number;
+};
+
+/**
+ * Those coordinates are represented in the coordinate space of the camera source (e.g. when you
```

Comment:
"those" refers to something at some distance away from us, so I think "these" would be a better choice here

```suggestion
 * These coordinates are represented in the coordinate space of the camera source (e.g. when you
```